docs/install/linux/index.md

```diff
@@ -265,237 +269,3 @@ windows) for older python versions. In the case that you need a
 different version than the one MPF requires, it is advised to create a
 python virtual environment and install the required packages there, and
 use that virtual environment for running MPF.
----
```

Comment:
Whew! Nice. We have too many installation instructions scattered around :P

Reply:
Once we have version depended mpf docs I can try to clean that up a bit more, I believe there is still room to improve on the install docs.